cmd/kubeadm/app/master/manifests_test.go

```diff
@@ -368,7 +339,7 @@ func TestGetAPIServerCommand(t *testing.T) {
 				Networking: kubeadm.Networking{ServiceSubnet: "bar"},
 			},
 			expected: []string{
-				"kube-apiserver",
+				"/usr/local/bin/kube-apiserver",
```

Comment:
see: https://github.com/kubernetes/kubernetes/issues/40248 and above https://github.com/kubernetes/kubernetes/pull/41400#discussion_r101380513